scripts/drone/promotion.star

```diff
@@ -7,10 +7,14 @@ def publish_gh_release():
         'image': 'cibuilds/github:0.13.0',
         'commands': [
             './scripts/generate_md5sum.sh',
+            '. ./scripts/get_gh_token.sh',
```

Comment:
nit: Noticed there's an extra `. ` here at the beginning of the line, not sure if its intentional.

Reply:
It's intentional, it's to make the exported GITHUB_TOKEN variable available for other scripts.